paperqa/prompts.py

```diff
@@ -53,6 +53,14 @@
     "Citation:"
 )
 
+structured_citation_prompt = (
+    "Extract the title, authors, and doi as a JSON from this MLA citation. "
+    "If any field can not be found, return them as null. "
+    "Use title, authors, and doi as keys, author's value should be a list of authors. "
+    "{text}\n\n"
```

Comment:
Can you change this template var name from `text`? Maybe `instructions_suffix`

Also a nit is if it's an empty string, there will be extra spaces

Reply:
renamed to "citation"